services/sts/src/main/java/software/amazon/awssdk/services/sts/auth/SessionCredentialsHolder.java

```diff
@@ -27,22 +27,48 @@
 @SdkInternalApi
 @ThreadSafe
 final class SessionCredentialsHolder {
```

Comment:
Do we even need this anymore, since the expiration is in the session credentials now?

Reply:
It will be removed in subsequent PRs.